core/src/main/res/values/arrays.xml

```diff
@@ -233,16 +233,6 @@
         <item>3</item>
     </string-array>
 
-    <string-array name="nav_drawer_feed_filter_values">
-        <item>0</item>
```

Comment:
Side note: Changing the preference values like this would usually lead to data loss. In this case, it is okay because the feature is not released to the public yet.